specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2019-11-01-preview/account.json

```diff
@@ -1106,8 +1106,8 @@
         },
         "type": {
           "description": "Gets the reference type property.",
-          "type": "string",
-          "readOnly": true
+          "default": "CollectionReference",
```

Comment:
The Swagger semantic of `default` is to describe what the server will do, if this information was NOT on the wire (this is oriented for bandwidth saving). Most SDK codegen will not send this value on the wire then. Please confirm this is what you mean.

If you mean that you want client SDK to send this value explicitly by default in absence if customer input, the extension name is `x-ms-client-default-value`

Reply:
Updated, thank you.